examples/binding/host-guest/yank.yaml

```diff
@@ -29,32 +28,47 @@ solvents:
     nonbonded_method: NoCutoff # Main definition of the nonbonded method
     implicit_solvent: OBC2 # Onufriev-Bashford-Case GBSA model, name is related to Implicit solvents in OpenMM
 
+  pme:
+    nonbonded_method: PME
+    switch_distance: 11*angstroms
+    nonbonded_cutoff: 12*angstroms
+    ewald_error_tolerance: 1.0e-4
+    clearance: 12*angstroms
+    positive_ion: Na+
+    negative_ion: Cl-
+
 # Define the systems: What is the ligand, receptor, and solvent we put them in
 systems:
-  host-guest:
-    # These names all use the names we defined previously
-    receptor: cucurbit
+  implicit:
+    receptor: CB7
     ligand: B2
     solvent: GBSA
     leap:
       parameters: leaprc.gaff2
 
+  explicit:
+    receptor: CB7
+    ligand: B2
+    solvent: pme
+    leap:
+      parameters: [oldff/leaprc.ff14SB, leaprc.gaff, frcmod.ionsjc_tip3p]
```

Comment:
Can you change the parameters to the following: 

`leaprc.protein.ff14SB, leaprc.gaff, leaprc.water.tip3p`